docs/drafts/developer.md

```diff
@@ -173,23 +154,39 @@ v0.33.1, built 2023-06-28
 
 Typically, you'll want to press the space bar to have it open the UI in your web browser.
 
-Shortly after starting, Tilt processes the `Tiltfile`, resulting in:
+---
+
+## Special Setup for MacOS
+
+Some additional setup is necessary on Macintosh computers to install and configure compatible tooling.
+
+### Install Homebrew and tools
+Follow the instructions to [installing Homebrew](https://docs.brew.sh/Installation) and then execute the following to install tools:
+
+```sh
+brew install bash gnu-tar gsed
+```
+
+### Configure your shell
+Modify your login shell's `PATH` to prefer the new tools over those in the existing environment. This example should work either with `zsh` (in $HOME/.zshrc) or `bash` (in $HOME/.bashrc):
+
+```sh
+for bindir in `find $(brew --prefix)/opt -type d -follow -name gnubin -print`
+do
+  export PATH=$bindir:$PATH
+done
+```
 
-- Building the go binaries
-- Building the images
-- Loading the images into kind
-- Running kustomize and applying everything except the Deployments that reference the images above
-- Modifying the Deployments to use the just-built images
-- Creating the Deployments
+---
 
-### Making code changes
+## How it works
```

Comment:
Should we link to some of our operator-controller documentation that goes into more detail on how it works?

Reply:
Yes, but I'm not sure if we have a good starting point doc for this. Would https://github.com/operator-framework/operator-controller/blob/adcf6e9a329756944f0b12a8a907cc125f8ea2d3/docs/Tasks/installing-an-extension.md work?

Reply:
I would argue that "How it works" should not be in this document. To understand this someone has to go through all of the docs or the architecture docs. Lets remove this section. As less is better for developer doc IMO.

Reply:
Yep pulled it. Done in 7f54af6c